### PR TITLE
Update add-view-transitions.mdx

### PR DESCRIPTION
**Pull request by alexnguyennz**
#### Description (required)

- Add/remove spaces, periods where needed
- v3.x > v4.x
- transition:animation > transition:animate
- Quotes for consistency in frontmatter examples

Also at [this section](https://docs.astro.build/en/tutorials/add-view-transitions/#update-scripts) it only mentions two events where there are now 5, may need an update?

### CODE REVIEW

**Comment by sarah11918** [comment]
Thanks, @alexnguyennz - Yes to that section! I would say something like...?

> There are several [events during client-side routing that you can listen for](/en/guides/view-transitions/#lifecycle-events), and fire events when they occur. The scripts in your project will now need to hook into two events to run at the right time during page navigation: [astro:page-load](https://docs.astro.build/en/guides/view-transitions/#astropage-load) and [astro:after-swap](https://docs.astro.build/en/guides/view-transitions/#astroafter-swap).

Something like that?

**Comment by alexnguyennz** [comment]
Yep that sounds good - I can leave that to you? Since that's actual content/writing vs little changes.

**Review by sarah11918 — APPROVED** [review]
Go Team!

File: src/content/docs/en/tutorials/add-view-transitions.mdx [hunk 1]
```diff
@@ -174,15 +174,15 @@ The steps below show you how to extend the final product of the Build a Blog tut
...
     
     View your site preview **at a smaller screen size**, and try to use the hamburger menu to navigate between pages. Notice that your menu will no longer work after the first page load.
 
 ### Update scripts
 
-With view transitions, some scripts may no longer re-run after page navigation like they do with full-page browser refreshes. There are two events during client-side routing that you can listen for, and fire events when they occur: [`astro:page-load`](/en/guides/view-transitions/#astropage-load) and [`astro:after-swap`](/en/guides/view-transitions/#astroafter-swap).
+With view transitions, some scripts may no longer re-run after page navigation like they do with full-page browser refreshes. There are several [events during client-side routing that you can listen for](/en/guides/view-transitions/#lifecycle-events), and fire events when they occur. The scripts in your project will now need to hook into two events to run at the right time during page navigation: [astro:page-load](https://docs.astro.build/en/guides/view-transitions/#astropage-load) and [astro:after-swap](https://docs.astro.build/en/guides/view-transitions/#astroafter-swap).
```
Comment (sarah11918) [inline src/content/docs/en/tutorials/add-view-transitions.mdx:185]:
```suggestion
With view transitions, some scripts may no longer re-run after page navigation like they do with full-page browser refreshes. There are several [events during client-side routing that you can listen for](/en/guides/view-transitions/#lifecycle-events), and fire events when they occur. The scripts in your project will now need to hook into two events to run at the right time during page navigation: [astro:page-load](/en/guides/view-transitions/#astropage-load) and [astro:after-swap](/en/guides/view-transitions/#astroafter-swap).
```

**Pull request merged**